### ISSUE TITLE
Opaque text on a transparent background

### ISSUE THREAD

**Issue by thatvertigo** [StaffEngineer/bevy_cosmic_edit#87]
Atm the `FillColor` attribute still blends alpha/opacity, so it's impossible to have opaque text on a transparent background.

**Comment by Dimchikkk** [comment]
<img width="408" alt="image" src="https://github.com/StaffEngineer/bevy_cosmic_edit/assets/111751109/31932c0d-e866-4779-bb23-c0b162ca967d">

In old version of plugin (the version `velo` is currently using) it was definitely possible to achieve opaque text on a transparent background  with `Color::None` as node background for CosmicSpriteBundle. More investigation is needed how to achieve this in current version, @bytemunch any ideas? :)

Basically setting `FillColor(Color::NONE)` makes background transparent but text is transparent too :)

**Comment by bytemunch** [comment]
Yeeeah that's my `draw_pixel` replacement isn't it. I'll mess with the alphas try make it work
